src/request-logger.js

```diff
@@ -1,8 +1,11 @@
 const R = require('ramda')
 const { pickProperties, stringify } = require('./utils')
+const { filterLargeProp, filterLargeUrl } = require('./utils')
 
-const buildRequestLog = propsToLog => req => {
+const buildRequestLog = (propsToLog, propMaxLength) => req => {
   const reqProps = pickProperties(req, propsToLog)
+  reqProps.body = filterLargeProp(reqProps, 'body', propMaxLength.body)
```

Comment:
Se `propMaxLength` for undefined, vai quebrar, não vai ?

Reply:
Não, retornaria o objeto original (até onde testei pelo menos)

Reply:
Se a variável `propMaxLength` for `undefined`, vai quebrar com o erro `Cannot read property 'body' of undefined`

Reply:
Essa variável `propMaxLength` vem direto da config que o escriba recebe, e não tem valor default. Então se não passar essa config pro escriba, aparentemente ele vai quebrar.

Reply:
ahhh verdade, da forma que tava antes estaria ok, mas agora que acesso uma prop realmente quebraria, boa!

Reply:
Aliás vou ajustar pra URL tbm, mesmo probleminha